include/exadg/postprocessor/spectral_analysis/bijection.h

```diff
@@ -206,9 +207,8 @@ class Bijection
         int temp_counter = counter;
         for(int d = 0; d < s.dim; d++)
         {
-          int r = temp_counter % n;
+          Y[d] = temp_counter % n;
           temp_counter /= n;
-          Y[d] = r + 0.5;
```

Comment:
Can you briefly explain why this change is legal?

Reply:
I was wondering about this one, too. My reasoning is that `r` is a non-negative integer, and `Y` is an array of integers. So I cannot see that `(int)(r + 0.5)` gives anything else than `r`, so one can simply cast that back. I did not try to really understand the algorithm here. Do we have a good test for this?

Reply:
Let's ask @peterrum ?

Reply:
@peterrum would you like to open an issue or pull request for that? I think we should keep this out of the present PR.